test/Interop/Cxx/class/Inputs/loadable-types.h

```diff
@@ -3,6 +3,10 @@
 
 struct EmptyStruct {};
 
+struct IntBox {
+  int x;
+};
+
```

Comment:
Can you add this struct declaration to `synthesized-initializers.h` instead? We try to keep the inputs of related tests in files with the same name.